tcms/search/__init__.py

```diff
@@ -38,10 +38,10 @@ def advance_search(request, tmpl='search/advanced_search.html'):
     all_forms = (plan_form, case_form, run_form)
     errors = [f.errors for f in all_forms if not f.is_valid()]
     if errors or not data:
-        PRODUCT_CHOICE = [
+        product_choice = [
```

Comment:
Rebase onto #308 or drop this commit.

Reply:
I dropped the commit since it is present in #308